packages/shared/src/components/search/MobileSearch.tsx

```diff
@@ -0,0 +1,104 @@
+import React, {
+  MouseEvent,
+  FormEventHandler,
+  ReactElement,
+  useEffect,
+  useRef,
+  useState,
+  FormEvent,
+} from 'react';
+import { useQuery } from 'react-query';
+import InteractivePopup, {
+  InteractivePopupPosition,
+} from '../tooltips/InteractivePopup';
+import { cloudinary } from '../../lib/image';
+import CloseButton from '../CloseButton';
+import { Button } from '../buttons/Button';
+import { SearchSubmitButton } from './SearchSubmitButton';
+import { SearchBarSuggestionList } from './SearchBarSuggestionList';
+import { generateQueryKey, RequestKey } from '../../lib/query';
+import { getSearchSuggestions } from '../../graphql/search';
+import { useAuthContext } from '../../contexts/AuthContext';
+import { SearchHistory } from './SearchHistory';
+
+interface MobileSearchProps {
+  input: string;
+  onSubmit(event: FormEvent, value: string): void;
+  onClose(event: MouseEvent, value: string): void;
+}
+
+export function MobileSearch({
+  input: initialInput,
+  onClose,
+  onSubmit,
+}: MobileSearchProps): ReactElement {
+  const { user } = useAuthContext();
+  const [input, setInput] = useState(initialInput);
+  const inputRef = useRef<HTMLInputElement>();
+  const { data, isLoading } = useQuery(
+    generateQueryKey(RequestKey.SearchHistory, user),
+    getSearchSuggestions,
+  );
+
+  const onSubmitForm: FormEventHandler = (event) => {
+    event.preventDefault();
+    onSubmit(event, input);
+  };
+
+  useEffect(() => {
+    inputRef.current?.focus();
+  }, []);
+
+  return (
+    <InteractivePopup
```

Comment:
Why did we not go for a modal, or is this the way we always do the overlapping ones?

Reply:
We currently don't have a modal configuration that has the same header. So thought of using this as it is an experiment and might change soon enough.